### PR TITLE
Add auto-insertion of 'host' param to API schema (+ small cleanup).

### PR DESCRIPTION
**Pull request by xor-xor**

### CODE REVIEW

File: src/ralph_scrooge/management/commands/generate_json_schema.py [hunk 1]
```diff
@@ -47,6 +47,14 @@ def handle(self, *args, **options):
                     "Aborting.".format(e)
                 )
                 return
+        if settings.SCROOGE_HOST:
```
Comment (szok) [inline src/ralph_scrooge/management/commands/generate_json_schema.py:50]: There will be better?
```
if not settings.SCROOGE_HOST:
    logger....
    return

schema['host'] = settings.SCROOGE_HOST
```

File: src/ralph_scrooge/settings/base.py [hunk 2]
```diff
@@ -366,8 +366,10 @@
 EMAIL_NOTIFICATIONS_SENDER = ''
 EMAIL_USE_TLS = False
 
-# Base URL component for constructing Scrooge's links in outgoing mails.
-BASE_MAIL_URL = ''
+# E.g. 'scrooge.mydomain.com', 'localhost:8000' - this setting will be used for
+# cases when you need to have an access to Scrooge's hostname, but you don't
+# have an acces to HTTP request or socket.
+SCROOGE_HOST = ''
```
Comment (ghost) [inline src/ralph_scrooge/settings/base.py:374]: I'd add note if this is required or optional

**Review by ghost — APPROVED** [review]

File: src/ralph_scrooge/management/commands/generate_json_schema.py (hunk 1 above)
Comment (xor-xor) [reply src/ralph_scrooge/management/commands/generate_json_schema.py:50]: That wouldn't handle the case where you'd have `host` specified directly/manually in schema file (which may be much more convenient during development).

File: src/ralph_scrooge/settings/base.py (hunk 2 above)
Comment (xor-xor) [reply src/ralph_scrooge/settings/base.py:374]: Done.

**Review by szok — APPROVED** [review]

**Pull request merged**